src/couch_replicator_tasks.erl

```diff
@@ -41,13 +41,13 @@ handle_call(all, _From, State) ->
             {id, iolist_to_binary([BaseID, Ext])},
             {source, iolist_to_binary(Source)},
             {target, iolist_to_binary(Target)},
+            {db, Rep#rep.db_name},
```

Comment:
Will this be a shard name? Wonder if we could/should aggregate any of these tasks by mem3:dbname() name. Customers might not care about.  It would probably be fabric/chttpd handling that